controllers/osbuildenvconfig_controller_test.go

```diff
@@ -886,40 +860,31 @@ var _ = Describe("OSBuildEnvConfig Controller", func() {
 													})
 												})
 
-												Context("ConfigMap for the ansible config exists", func() {
+												Context("Worker API Service exists", func() {
 													const (
-														workerOSBuildWorkerConfigConfigMapName = "osbuild-worker-config"
-													)
-
-													var (
-														workerConfigAnsibleConfigConfigMap = corev1.ConfigMap{
-															ObjectMeta: metav1.ObjectMeta{
-																Namespace: operatorNamespace,
-																Name:      workerConfigAnsibleConfigConfigMapName,
-															},
-														}
+														workerConfigAnsibleConfigConfigMapName = "osbuild-worker-setup-ansible-config"
```

Comment:
Why not to use the const from the controller?

Reply:
I can't use the const from the controller because the test is in a different package so it does not have access to internal variables and consts. I could expose them, but, I think it's bad practice.
Let's say someone accidentally changes the value of a const (typing a key without noticing where the cursor is), using the same const will not catch it, while separately defining the expected value will.

Reply:
Yes I meant to expose them but if you prefer not to I get it.